{"componentChunkName":"component---node-modules-lekoarts-gatsby-theme-minimal-blog-core-src-templates-homepage-query-tsx","path":"/","result":{"data":{"allPost":{"nodes":[{"__typename":"MdxPost","slug":"/python-기초-문법","title":"Python 기초 문법","date":"13.01.2021","excerpt":"식별자와 스타일 가이드 Python은 동적 타이핑 언어로 Java…","timeToRead":2,"description":"Python 기초 문법","tags":[{"name":"Python","slug":"python"}]},{"__typename":"MdxPost","slug":"/python-class","title":"Python Class","date":"13.01.2021","excerpt":"…","timeToRead":1,"description":"Python Class","tags":[{"name":"Python","slug":"python"}]},{"__typename":"MdxPost","slug":"/python-예외처리","title":"Python 예외처리","date":"13.01.2021","excerpt":"예외처리 프로그래밍을 하다 보면 수 없이 많은 오류와 마주하게 된다. 오류가 발생하면 오류가 발생한 줄 아래로는 실행이 되지 않는데 때로는 오류가 발생하더라도 실행해야 할 때가 있다. 그럴 때 사용하기 위한 것이 예외처리 기법이다. try…","timeToRead":1,"description":"Python 예외처리","tags":[{"name":"Python","slug":"python"}]}]}},"pageContext":{"formatString":"DD.MM.YYYY"}},"staticQueryHashes":["3090400250","3090400250","3090400250","318001574"]}